spinnaker-monitoring-daemon/requirements-docker.txt

```diff
@@ -0,0 +1,21 @@
+# sudo apt-get install python-dev
```

Comment:
I'm not a fan of duplicating a critical config file - two files to maintain invariably get out of sync. I'd prefer doing some magic in the Dockerfile to enable all providers.  A simple way would be to append a unique comment string before each optional dependency. I use`#@` as an arbitrary example below:

requirements.txt
```
# A dependency that starts with #@ will be auto-enabled in the Docker container.
#@datadog
#@ google-api-python-client
#@       oauth2client
#@ prometheus_client
```

Dockerfile:
```
COPY requirements.txt /opt/spinnaker-monitoring/requirements.txt

RUN sed -i "s/#@[[:space:]]*//g" requirements.txt
```

For some reason, I couldn't get `\s` to work in sed on OS X, hence `[[:space:]]`. This should produce the proper `requirements.txt` file:

```
# A dependency that starts with #@ will be auto-enabled in the Docker container.
datadog
google-api-python-client
oauth2client
prometheus_client
```